lib/susepubliccloudinfoclient/infoserverrequests.py

```diff
@@ -297,6 +297,11 @@ def __process(url, info_type, command_arg_filter, result_format):
     """
     server_response_data = __get_data(url)
     resultset = __parse_server_response_data(server_response_data, info_type)
+    if info_type == 'regions':
+        if not resultset:
+            no_region_msg = "No region information available. Images have " \
+                            "the same identifier in all regions"
```

Comment:
Is there a case where there are just no region identifiers at all? I think the message here does match what was in the original service ticket, I'm just checking if its accurate.

Reply:
Yes, in Oracle, we only have BYOS images, i.e. no update infrastructure, and images are global. Based on the XML data there is no way for the API on the server side to derive any region information.

Reply:
I strongly disagree with returning a document like:

```
<regions><region>No region information available. Images have the same identifier in all regions.</region></regions>
```

That just feels wrong to me. Now, as a client author, I can't trust that the list of regions coming back is actually that, I have to look for some magic string and handle the case of "if there's only 1 region and it says blahblahblah then that's not a region".

I would _strongly_ prefer we send an empty list, and use an HTTP response code, and a message in a different context to indicate that.

This could be as simple as returning `204 NO CONTENT`, and optionally a well-formed document: `<info>No region information available.</info><regions></regions>`.

Reply:
In other words, as a client of the API, if I ask for a list of regions, and get a 200 OK response; it _better_ include a list of regions, even if it's an empty list.

Reply:
AFAICT, this PR merely deals with *displaying* the information on the screen. Are you saying there are tooling out there that actually parse the stdout as part of an overall pipeline?

Reply:
On the protocol side, I agree with returning HTTP 204 if filter yield no results.

Reply:
OH DUH. Never mind.